rpcs3/Emu/RSX/VK/vkutils/sampler.cpp

```diff
@@ -25,7 +62,20 @@ namespace vk
 		info.minFilter = min_filter;
 		info.mipmapMode = mipmap_mode;
 		info.compareOp = depth_compare_mode;
-		info.borderColor = border_color;
+		info.borderColor = border_color.value;
+
+		VkSamplerCustomBorderColorCreateInfoEXT custom_color_info;
```

Comment:
doesn't this warn about initialization?

Reply:
Nope. The compiler is smart enough to see it is initialized before use.

Reply:
"The compiler". Which one xD